lib/Gemfile

```diff
@@ -76,5 +77,9 @@ unless ENV['APPLIANCE']
     gem "rspec-fire", "~>1.3.0",  :require => false
     gem "timecop",    "~>0.5.3",  :require => false
     gem "xml-simple", "=1.0.12",  :require => false  # Used by test/xml/tc_xmlhash_methods.rb
+    gem "vcr",        "~>2.6",    :require => false  # Used by manageiq-foreman
+    gem "webmock",                :require => false  # Used by vcr / manageiq-foreman
```

Comment:
I get this warning locally:

```
Your Gemfile lists the gem vcr (~> 2.6) more than once.
You should probably keep only one of them.
While it's not a problem now, it could cause errors if you change the version of just one of them later.
Your Gemfile lists the gem webmock (>= 0) more than once.
You should probably keep only one of them.
While it's not a problem now, it could cause errors if you change the version of just one of them later.
```

If they are moving here, can you remove them from vmdb Gemfile as well?